### ISSUE TITLE
Parallel pipeline steps show wrong name in display

### ISSUE THREAD

**Issue by DevInsanity** [jan-molak/jenkins-build-monitor-plugin#380]
When running a Jenkins pipeline job with parallel steps, the build monitor display shows the stage name for each step, rather than the branch name of the step.

For example, the pipeline script:
```
stage('test-stage') {
    parallel a: { -> sleep 60 }, b: { -> sleep 60 }, c: { -> sleep 60 }
}
```
while running, the build monitor shows "[test-stage, test-stage, test-stage]" next to the build number, rather than the expected "[a, b, c]" or even just the stage name once.

**Comment by ChadiEM** [comment]
Hey @DevInsanity, thanks for reporting this!

It reminds me of #329, which has been fixed with PR #334. Can you please check that your build monitor plugin is up to date?

**Comment by DevInsanity** [comment]
@ChadiEM You're correct. Unfortunately, the Jenkins instance on which this was spotted isn't fully up to date, and it seems my searching wasn't good enough to spot the previous issue.
